crates/pallet-subspace/src/extensions/mod.rs

```diff
@@ -0,0 +1,186 @@
+//! Extensions for unsigned general extrinsics
```

Comment:
Most of our modules use the modern Rust module layout (`name.rs`), rather than the legacy layout (`name/mod.rs`).
Is there any particular reason we’re using the legacy layout here?

Reply:
I wanted to keep the extension specific weights and benchmarking under its own module rather than polluting the main module.

Reply:
But the legacy and modern module file layouts can both do that. So I just want to make sure we are talking about the same thing here?

In Subspace we use `extensions.rs` rather than `extensions/mod.rs`. `mod.rs` is an older layout:
https://doc.rust-lang.org/book/ch07-05-separating-modules-into-different-files.html#alternate-file-paths

Can you rename this file to `src/extensions.rs`, to be consistent with the rest of the codebase? (Nothing else needs to change.)